### ISSUE TITLE
How to use `Automatically Extend Workspace Makefile` feature?

### ISSUE THREAD

**Issue by quake** [sagiegurari/cargo-make#96]
#### Problem Description
I want to use `Automatically Extend Workspace Makefile` feature, however, it does not work as expected. 

I'm using `examples/workspace2` as scaffold, add env to root Makefile.toml:
```
[env]
CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = "true"
```

delete `Makefile.toml` from `member2` folder.

run `cargo make echo`, got this error:

```
[cargo-make] INFO - cargo-make 0.11.1
[cargo-make] INFO - Using Build File: Makefile.toml
[cargo-make] INFO - Task: echo
[cargo-make] INFO - Setting Up Env.
[cargo-make] INFO - Adding Member: member2.
[cargo-make] INFO - Adding Member: member/member3.
[cargo-make] INFO - Running Task: init
[cargo-make] INFO - Running Task: workspace
+ cd /Users/quake/workspace/cargo-make/examples/workspace2
+ cd ./member2
+ cargo make --disable-check-for-updates --no-on-error --loglevel=info echo
[cargo-make] INFO - cargo-make 0.11.1
[cargo-make] INFO - Using Build File: Makefile.toml
[cargo-make] INFO - Task: echo
[cargo-make] INFO - External file not found, skipping.
[cargo-make] INFO - Setting Up Env.
[cargo-make] ERROR - Task not found: echo
[cargo-make] WARN - Build Failed.
[cargo-make] ERROR - Error while executing command, exit code: 1
[cargo-make] WARN - Build Failed.
```

Am I doing anything wrong or stupid?

**Comment by sagiegurari** [comment]
seems I had some regression bug there. will release a fix soon.

**Comment by sagiegurari** [comment]
fix published in version 0.11.2
please confirm it works good for you.

**Comment by quake** [comment]
It works, thank you!
